modules/dnn/test/test_darknet_importer.cpp

```diff
@@ -353,7 +353,7 @@ TEST_P(Test_Darknet_nets, YoloVoc)
                                     1, 6,  0.844947f, 0.637058f, 0.460398f, 0.828508f, 0.66427f);  // a car
 
     double nmsThreshold = (target == DNN_TARGET_MYRIAD) ? 0.397 : 0.4;
-    double scoreDiff = 8e-5, iouDiff = 3e-4;
+    double scoreDiff = 1e-4, iouDiff = 3e-4;
```

Comment:
The arm instruction `vfmaq_laneq_f32` is only supported at ARMv8. So I use the `vmlaq_n_f32` instead. I adjusted the thresholds here and below because I found the `vmlaq_n_f32` will generate very little different results than `vmlaq_n_f32`.